lessons/Data-Science/histograms-interpret/langs/en-us/index.adoc

```diff
@@ -0,0 +1,323 @@
+
+@teacher{
+The third section of the posters will remain blank today. Students will complete this section in the future when they learn about @lesson-link{box-plots}.
+}
+
+
+@lesson-instruction{
+Try it on your own! Complete @printable-exercise{reading-histograms-matching.adoc}, where you will match descriptions of video ratings to histograms that could fit the data.
+}
+
+@teacher{Go over correct responses as a class.}
+
+
+=== Synthesize
+
+@QandA{
+@Q{What strategies did your group use in brainstorming scenarios to match histogram shape?}
+@Q{For which distributions was it easiest to come up with an example?}
+@Q{For which distributions was it hardest to come up with an example?}
+}
+
+
+== Histograms and Measures of Center
+
+@objective{histograms-and-measures-of-center}
+
+=== Overview
+
+Students combine apply their knowledge histograms, measures of center, and shape.
+
+=== Launch
+
+Can you compute and interpret different measures of center from a histogram? Let's try it.
+
+@QandA{
+The histogram below shows the number of children per home in the Broadmoor neighborhood.
+
+@center{@image{images/number-children.png, 250}}
+
+@Q{What is the median number of children per home?}
+@A{The median number of children per home is 1.}
+@A{Strategy 1: List out the raw data (0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 4, 4, 5, 7, 8) then locate the middle value.}
+@A{Strategy 2: Determine the total number of homes represented by adding columns' heights (6 + 8 + 4 + 3 + 2 + 1 + 1 + 1 = 26), then locate the 13th and 14th values on the histogram.}
+@A{Students may mistakenly attempt to find the midpoint of the values on the horizontal axis (4.5), indicating that they connect median with “middle”, but misunderstand _what_ middle value to find.}
+
+@Q{What is the mean number of children per home?}
+@A{Approximately 2.04. Note that students may attempt to use the “add up and divide” algorithm with inappropriate data values from the display. Students may mistakenly compute the mean height of the bars, or the mean of values on the horizontal axis.}
+}
+
+@teacher{Students often cannot compute measures of center from histograms because they _lack attention to the context of the data_. Cooper and Shore (2008) suggest that when students are confused, simply ask "What are the data?" to help reorient and redirect students. Similarly, urge students to discuss and defend their responses.}
+
+
+=== Investigate
+
+@lesson-instruction{
+- Work with a partner to complete @printable-exercise{histograms-moc-1.adoc}.
+- Which problem was the most challenging? Why did you find it challenging?
+}
+
+@teacher{
+Discuss solutions with students. Highlight for students the misconceptions revealed by common incorrect responses. For instance, choosing "the modes are roughly the same" for the the first problem suggests that students understand the concept of mode, but looked at bar height (indicating frequency) instead of considering the value on the x-axis.
```

Comment:
If we are instructing teachers to discuss misconceptions we need to list them here. Are you referring to the ones mentioned in the previous lesson? (If so, copy them over.) Are there others in your head, @retabak?